src/components/HomeContent/UpcomingMovies.jsx

```diff
@@ -140,6 +150,7 @@ const UpcomingMovies = () => {
                       className="flex flex-col w-[320px] h-80 border rounded-sm cursor-pointer hover:scale-110 duration-300 ease-out hover:drop-shadow-xl hover:shadow-3xl upcmvs__crd__cntr"
                       onMouseEnter={() => handleMouseEnter(index)}
                       onMouseLeave={handleMouseLeave}
+                      onClick={() => handleClick(item.id)}
```

Comment:
Why not use `Link` Element

Reply:
what problem in which ?
<img width="952" alt="Screenshot 2024-06-15 at 6 36 17 PM" src="https://github.com/tsbridgeedu/filmskart/assets/149366729/ceed9fcd-1261-4c63-b4b9-e2a9778b573d">